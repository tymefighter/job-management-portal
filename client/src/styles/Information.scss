.information {
    display: flex;
    flex-direction: column;
    gap: 2rem;
    padding: 0 3rem;
}

.information__heading {
    font-size: 2.5rem;
    text-align: center;
}

.information__item-heading {
    font-size: 1.7rem;
    font-style: italic;
}

.information__item-desc, .information__item-list {
    font-size: 1.5rem;
}