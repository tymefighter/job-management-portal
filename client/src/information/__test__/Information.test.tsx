import { render } from "@testing-library/react";
import Information from "../Information";

beforeEach(() => {
    document.title = "";
})

test("Information snapshot test", () => {
    const {container} = render(<Information />);

    expect(container.firstChild).toMatchSnapshot("Information Component Snapshot");
});

test("Information title test", () => {
    expect(document.title).toEqual("");

    render(<Information />);
    expect(document.title).toEqual("Information");
});