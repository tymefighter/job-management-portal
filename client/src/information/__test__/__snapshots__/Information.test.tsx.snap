// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Information snapshot test: Information Component Snapshot 1`] = `
<div
  class="information"
>
  <h1
    class="information__heading"
  >
    Information
  </h1>
  <div
    class="information__item"
  >
    <h2
      class="information__item-heading"
    >
      Job Management Portal
    </h2>
    <p
      class="information__item-desc"
    >
      The Job Management Portal is an application which allows it's users to store information about companies, the comments that people have about them and the jobs that are being offered by the company. It also allows the users to modify the data that is present by editing current information such as Company logo, name, jobs, a comment about it, etc, and adding more information such as companies, jobs and comments.
    </p>
  </div>
  <div
    class="information__item"
  >
    <h2
      class="information__item-heading"
    >
      Our Vision
    </h2>
    <p
      class="information__item-desc"
    >
      Our vision is to provide the users with an application which helps them manage easily information about companies and job opportunities, this would ultimately help them in focussing on what is more important, such as interview preparation, competition information and more.
    </p>
  </div>
  <div
    class="information__item"
  >
    <h2
      class="information__item-heading"
    >
      Our Motivation
    </h2>
    <p
      class="information__item-desc"
    >
      Our motivation into building this application was that we want every human to never miss an important opportunity to attain success. We want people to focus on what is important, and not what can be automated. We want humans to be human, and the rest to be handled by automation.
    </p>
  </div>
  <div
    class="information__item"
  >
    <h2
      class="information__item-heading"
    >
      Our Values
    </h2>
    <ul
      class="information__item-list"
    >
      <li>
        Bring Forth Opportunities
      </li>
      <li>
        Improvise Productivity
      </li>
      <li>
        Strength in Information
      </li>
      <li>
        Technology is the Key Solution
      </li>
      <li>
        Human Upliftment
      </li>
    </ul>
  </div>
</div>
`;
